src/pages/news/SingleNews.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { Button } from '@/components/Button'
+import ChangeColor from '@/components/IconSVG/ChangeColor'
+
+const SingleNews = ({
+  title,
+  description,
+  imageUrl,
+  index,
+}: {
+  title: string
+  description: string
+  imageUrl: string
+  index: number
+}) => {
+  return (
+    <div className="container">
```

Comment:
Why we are not using shared Container component for defining this kind of structure, we should have component like this 

`export const Container = (children) => {
 return (
    <div className="container">
        {children}
     </div>
 )
}`

Reply:
Done